app/main.py

```diff
@@ -1,3 +1,15 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> None:
+    list_of_results = {}
+
+    def inner(*args: Any, **kwargs: Any) -> None:
```

Comment:
Wrong return type!
Also no need to check for instance in this task no needed

Reply:
it must be ` -> Callable`

Reply:
because it is decorator

Reply:
pls do this